models/beers.js

```diff
@@ -0,0 +1,38 @@
+var keystone = require('keystone'),
+  Types = keystone.Field.Types;
+
+/**
+ * Beers Model
+ * ==========
+ */
+
+var Beers = new keystone.List('Beers', {
+  map: { name: 'beerName' },
+  autokey: { path: 'slug', from: 'beerName', unique: true }
+});
+
+Beers.add(  
+  {heading: 'Beer'},
+    {
+      beerTitle: { type: String },
+      beerName: { type: String, required: true, initial: true},
+      brewerName: { type: String, required: true, initial: true},
+      beerImage: {type: Types.CloudinaryImage, required: false, initial: false },
+      isFeaturedBeer: { type: Types.Boolean, required: false}
+    },
+  {heading: 'Beer Profile'},
+    {
+      profileTitle: { type: String },
+      profileAuthor: { type: Types.Name },
+      profileBody: { type: Types.Textarea },
+      flavorProfile: { type: Types.Select, options: 'Malty, Earthy, Aromatic, Sweet, Rich, Delicious', default: 'Delicious', index: true },
```

Comment:
So, talk to Anders about this, I believe the flavor profile will probably be more complex than a single item. Find out what he wants to track in terms of flavor profile for v1, hit him on Slack